workflow-templates/im-test-playwright-manual.yml

```diff
@@ -0,0 +1,224 @@
+          | Branch/Tag              | `${{ env.GITHUB_REF }}`                                                                   |
+          | Workflow Branch/Tag     | `${{ github.ref_name }}` - SHA: `${{ github.sha }}`                                       |
+          | Browser                 | `${{ inputs.browser-project }}`                                                           |
+          | Playwright Config File  | `${{ inputs.config-file }}`                                                               |
+          | Test Filter             | `${{ inputs.test-filter != 0 && inputs.test-filter || 'NA' }}`                            |
+          | Send Teams Notification | `${{ inputs.send-teams-notification == 0 && 'false' || inputs.send-teams-notification }}` |' >> $GITHUB_STEP_SUMMARY
+
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.GITHUB_REF }}
+
+      - name: Set Cache Keys
+        # As long as these files don't change the cache will remain in tact and the time
+        # to restore the different packages in the various jobs will be reduced.  If one
+        # of the files does change though, a new cache will be uploaded the next time
+        # it runs, then subsequent runs should be faster.
+        run: echo "NPM_CACHE_KEY=playwright_node_modules-${{ hashFiles('**/tests/playwright/package-lock.json') }}" >> $GITHUB_ENV
+
+      - name: Check for an npm cache
+        id: has-cache
+        uses: actions/cache@v4
+        with:
+          lookup-only: true
+          key: ${{ env.NPM_CACHE_KEY }}
+          path: |
+            ${{ env.NPM_CACHE_PATH }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Authenticate with GitHub Packages
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        uses: im-open/authenticate-with-gh-package-registries@v1
+        with:
+          read-pkg-token: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
+          orgs: 'im-client,im-enrollment,im-customer-engagement,im-practices,im-open'
+
+      - name: NPM Install
+        if: steps.has-cache.outputs.cache-hit != 'true'
+        working-directory: '${{ env.PLAYWRIGHT_TEST_DIR }}'
+        run: |
+          echo "Installing NPM"
+          npm ci
+
+      # The remaining steps will only be executed if the cache was not found, otherwise they will be skipped.
+      # This action creates a post-job step that will upload the node_modules dir to the cache if the job
+      # completes successfully.  Subsequent jobs and workflow runs can use this cached version of the node_modules
+      # folder if the package-lock.json hasn't changed and it uses a ubuntu-20.04 runner to restore the cache from.
```

Comment:
Is this comment block supposed to be over all of the steps with the `if: steps.has-cache.outputs.cache-hit != 'true'` condition?

Reply:
Moved to above first step that has the if statement.